### PR TITLE
Improve key rotation section

### LINKED ISSUE #19166: Improve key rotation section
### Description

Add a section to the Server Administration Guide on rotating keys in the realms chapter

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Closes #19166

This replaces https://github.com/keycloak/keycloak-documentation/pull/1750 which had to moved to the new repository.

@ikhomyn Can you approve this again and @abstractj can you check if this works.  I changed the KeyPair terms and removed one instance of wildfly.

### CODE REVIEW

File: docs/documentation/server_admin/topics/realms/keys.adoc [hunk 1]
```diff
@@ -22,35 +22,78 @@ that is able to provide an active keypair.
...
++
+[Source,Bash,Subs=+Attributes]
+----
+keytool -delete -keystore <keystore>.jks -storepass <password> -alias <key>
+----
+. Import the new certificate into the keystore
++
+[Source,Bash,Subs=+Attributes]
+----
+keytool -importcert -file domain.crt -keystore <keystore>.jks -storepass <password>  -alias <key>
```
Comment (ikhomyn) [inline docs/documentation/server_admin/topics/realms/keys.adoc:88]:
```suggestion
keytool -importcert -file domain.crt -keystore <keystore>.jks -storepass <password> -alias <key>
```

**Review by ikhomyn — APPROVED** [review]
Hi @andymunro, looks good

**Comment by andymunro** [comment]
@ikhomyn Can you approve this again and @abstractj can you check if this works. I changed the KeyPair terms and removed one instance of wildfly.

**Review by ikhomyn — DISMISSED** [review]
Hi, @andymunro Looks good.

**Comment by abstractj** [comment]
@andymunro submitted my suggested changes on top of yours. Could you please review?

**Review by andymunro — COMMENTED** [review]
looks good

**Review by abstractj — APPROVED** [review]

**Pull request merged**